Soccar [Game-Server]/src/nl/soccar/gameserver/message/MovePlayerMessage.java

```diff
@@ -1,6 +1,7 @@
 package nl.soccar.gameserver.message;
 
 import nl.soccar.gameserver.message.handler.MovePlayerMessageHandler;
+import nl.soccar.library.Player;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.library.Player'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)